### PR TITLE
feat(FEC-10666): add HD/4K indication in quality selector

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

We wish to add a simple indication that would allow millennials to know the quality they are watching.

Requirements
For resolutions, where height is between 720 (included) and 2160 (excluded) add an HD indication.
For resolutions, where height is between 2160 (included) and 4320 (excluded) add a 4K indication.
For resolutions, where height is above 4320 (included) add a 8K indication.

solves FEC-10666

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated
Screen Shot 2021-07-29 at 12 53 51

updated: with svg icons

Screen Shot 2021-08-08 at 10 39 03

### CODE REVIEW

File: src/components/dropdown/dropdown.js [hunk 1]
```diff
@@ -120,31 +120,20 @@ class DropDown extends Component {
...
-   * get active option label or first option's label
+   * get active option or first option
    *
-   * @returns {string} - active option label
+   * @returns {Object} - active option
    * @memberof DropDown
    */
-  getActiveOptionLabel(): string {
-    let activeOptions = this.props.options.filter(t => t.active);
+  getActiveOptionLabel(): Object {
```
Comment (RoyBregman) [inline src/components/dropdown/dropdown.js:128]: rename

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/components/smart-container/_smart-container.scss [hunk 2]
```diff
@@ -82,7 +82,6 @@
       .dropdown {
         span {
           max-width: 100px;
-          overflow: hidden;
```
Comment (yairans) [inline src/components/smart-container/_smart-container.scss:85]: need to support long label (FEC-7236)

File: src/styles/_dropdown.scss [hunk 3]
```diff
@@ -41,6 +54,12 @@
       transition: 150ms transform;
       will-change: transform;
     }
+
+    .label-badge {
```
Comment (yairans) [inline src/styles/_dropdown.scss:58]:
```suggestion
    .label-badge.badge-icon {
```

File: src/styles/_dropdown.scss [hunk 4]
```diff
@@ -113,5 +133,11 @@
     .menu-icon-container {
       opacity: 0;
     }
+
+    .label-badge {
+      &[class*='badge-icon'] {
```
Comment (yairans) [inline src/styles/_dropdown.scss:138]: same

**Review by yairans — APPROVED** [review]

**Review by yairans — APPROVED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**